react-file-uploads/client/app.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    fetch('/api/uploads', {
+      method: 'post',
+      body: formData
+    })
+      .then(res => res.json())
+      .then(res => console.log(res))
+      .then(event.target.reset())
```

Comment:
Be careful to use a function _definition_ here. Otherwise, the form will get reset immediately, instead of waiting.

```js
.then(() => event.target.reset())
```